### ISSUE TITLE
Write cron script to hop passwords

### ISSUE THREAD

**Issue by jaygoodswp** [jaygoodswp/PHS#57]

**Comment by TMM21042** [comment]
HI Jay;  This can be done in two ways. One is a robotic method which requires a unique script for each site. The second way is via an API. I think the second way is preferred for several reasons. But not all external sites  May be capable of using an API. If we use an API, then we also want to create or have an option two disable email notifications of account password changes originating from the external site, as well as an option to preserve the user account login to any site app.



Sent from my mobile device with auto complete technology. Please forgive incorrect grammar.

---- jaygoodswp wrote ----

**Comment by jaygoodswp** [comment]
This has been completed except for actually changing the password on the sites.  As we have discussed, this will require a plan of action.

**Comment by jaygoodswp** [comment]
I agree with what you're saying but feel that we need to come up with a game plan.  It's going to be really hard to convince a website to give us API access in order to change a user account.  Even if we impersonate the user by logging on to the API with their account, there is not likely to be functions that change the password.  I realize that you'd like me to write up a specification for the API that a website should provide to us, but getting them to actually write this API will be a huge challenge.  Let's have a chat and decide what we're going to do initially for folks to be able to hop their passwords.  The scripts that you provided will work for those few sites but obviously, we'll need to write one for each site that users want to use.

**Comment by jaygoodswp** [comment]
@TMM21042 The script to hop passwords is written and it works.  Also, the API for sites to use to authenticate a user is written and has been tested.  I'm going to mark this as completed.  We can then add issues if you'd like certain sites to be changed robotically.
